tenant/pkg/util/tenant_utils.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
Can we come up with anything better than `util`? Do we expect to add anything else here over time? If not, no worries.

Reply:
How about tenantutil?

Reply:
Maybe `pkg/api/tenantnamespace`? See https://github.com/kubernetes/kubernetes/blob/master/pkg/api/persistentvolume/util.go for reference.

Reply:
done